source/content/restore-environment-backup.md

```diff
@@ -87,3 +87,37 @@ This is a destructive process. If you're not comfortable with this technique, us
 5. Deploy the change from Dev up to Test and Live.
 
 For more information, see [git reset](https://git-scm.com/docs/git-reset).
+
+## Restoring the LIVE environment
+The restore process removes any recent content or changes applied to your site, since the date of the backup file you will use. Doing so in LIVE means you will lose them forever with no way to recover.
+
+If there is still a need to do this in the LIVE environment, we recommend doing the following steps:
+- [Lock the environment first](/docs/security/)
+- [Run a backup](/docs/backups/) of LIVE, in order to have a copy of files and the database for later use
```

Comment:
I assume you would only do this to get any content not including in older backups? If so we can mention that on L99.

Reply:
Ah right, I agree